internal/api/lib_test.go

```diff
@@ -61,6 +61,7 @@ func TestInitCacheEmptyCapabilities(t *testing.T) {
 	defer os.RemoveAll(tmpdir)
 	cache, err := InitCache(tmpdir, "", TESTING_CACHE_SIZE, TESTING_MEMORY_LIMIT)
 	ReleaseCache(cache)
+	require.NoError(t, err)
```

Comment:
I guess we want to check the error first, right?

```suggestion
	require.NoError(t, err)
	ReleaseCache(cache)
```